apps/nextjs-website/src/app/[...slug]/page.tsx

```diff
@@ -14,7 +14,7 @@ export const generateStaticParams = async (): Promise<Page[]> =>
   // "generateStaticParams"; __return_type__: Promise<readonly Page[]>; }' does
   // not satisfy the constraint '{ __tag__: "generateStaticParams";
   // __return_type__: any[] | Promise<any[]>; }'.
-  [...(await getAllPages())];
+  [...(await getAllPages())].filter((page) => page.slug[0] !== 'homepage');
```

Comment:
I would suggest implementing this logic on [makePageListFromNavigation](https://github.com/pagopa/b2b-portals/pull/141/files#diff-e2c23e92b60777da2aca3272942d1f18a2e966f25fa326eb54a5f7c610030ad6R25-R34) to facilitate testing and maintain application logic cohesive

Reply:
That would prevent homepage data from ever being rendered.

We still want getPageProps (hence getAllPages, hence makePageListFromNavigation) to be able to access homepage data by specifying the correct URL ('homepage')

If we were to move this logic, we would also need to create a separate function, akin to getAllPages, to only fetch the homepage, which does not seem worth it at all.

Reply:
Mentioning PR #94 as this could be one thing to keep in mind when choosing whether or not to implement it.